projects/samples/contests/robocup/worlds/kid.wbt

```diff
@@ -8,8 +8,11 @@ WorldInfo {
   basicTimeStep 16
 }
 Viewpoint {
-  orientation 0.3104147445196029 0.47491052518891463 0.8234699019694763 2.1540489988954423
-  position 10.605734337906961 4.537966315195794 6.8277591846650605
+  #orientation 0.3104147445196029 0.47491052518891463 0.8234699019694763 2.1540489988954423
+  orientation 0.7740618353667722 -0.3393112407167443 -0.5345055256514077 1.3737629675929948
+  #position -10.605734337906961 -4.537966315195794 6.8277591846650605
+  position -10.918806145468938 -9.442194213986694 6.060809393740457
```

Comment:
You shouldn't put comments here, but remove these lines.

Reply:
Thank you I forgot those, now it's corrected